### PR TITLE
Seabeta 537 back links

### PR DESCRIPTION
**Pull request by DanCorderSoftwire**
Add back links back to recommendation and action plan pages

### CODE REVIEW

File: SeaPublicWebsite/Views/StaticPages/AccessibilityStatement.cshtml [hunk 1]
```diff
@@ -1,18 +1,7 @@
...
     ViewBag.Title = $"Accessibility statement on {Constants.SERVICE_NAME}";
 }
 
-@section BeforeMain {
-    @await Html.GovUkBackLink(new BackLinkViewModel
-    {
-        Text = "Back",
-        OverrideWithJavascript = true
-    })
-}
```
Comment (DanCorderSoftwire) [inline SeaPublicWebsite/Views/StaticPages/AccessibilityStatement.cshtml:14]: This link shouldn't be here as it relies on JS

**Comment by DanCorderSoftwire** [comment]
It's not as bad as it looks, there's a lot of renaming going on

File: SeaPublicWebsite/Models/EnergyEfficiency/ActionPlanViewModel.cs [hunk 2]
```diff
@@ -5,10 +5,11 @@
 
 namespace SeaPublicWebsite.Models.EnergyEfficiency
 {
-    public class YourSavedRecommendationsViewModel
+    public class ActionPlanViewModel
     {
         public PropertyData PropertyData { get; set; }
         public string EmailAddress { get; set; }
+        public string BackLink { get; set; }
```
Comment (CodrinCotarlanSoftwire) [inline SeaPublicWebsite/Models/EnergyEfficiency/ActionPlanViewModel.cs:12]: You could inherit `QuestionFlowViewModel` instead of adding a backlink field

Comment (DanCorderSoftwire) [reply SeaPublicWebsite/Models/EnergyEfficiency/ActionPlanViewModel.cs:12]: But it's not part of the question flow :)

File: SeaPublicWebsite/Models/EnergyEfficiency/RecommendationViewModel.cs [hunk 3]
```diff
@@ -8,51 +8,30 @@ namespace SeaPublicWebsite.Models.EnergyEfficiency
...
-            }
-
-            return null;
+            return RecommendationIndex < PropertyRecommendations.Count - 1;
         }
 
         public List<PropertyRecommendation> GetSavedRecommendations()
         {
-            return PropertyData.PropertyRecommendations.Where(r => r.RecommendationAction == BusinessLogic.Models.Enums.RecommendationAction.SaveToActionPlan).ToList();
+            return PropertyRecommendations.Where(r => r.RecommendationAction == BusinessLogic.Models.Enums.RecommendationAction.SaveToActionPlan).ToList();
```
Comment (CodrinCotarlanSoftwire) [inline SeaPublicWebsite/Models/EnergyEfficiency/RecommendationViewModel.cs:32]: Can we import `BusinessLogic.Models.Enums.RecommendationAction.SaveToActionPlan` so we only write `SaveToActionPlan` here?

**Review by CodrinCotarlanSoftwire — APPROVED** [review]
Minor comments, looks good

File: SeaPublicWebsite/Models/EnergyEfficiency/ActionPlanViewModel.cs (hunk 2 above)
Comment (CodrinCotarlanSoftwire) [reply SeaPublicWebsite/Models/EnergyEfficiency/ActionPlanViewModel.cs:12]: True...

**Pull request merged**